### DISCUSSION TITLE
Why Polyfill JS is not added to the page's scripts

### DISCUSSION THREAD

**Discussion by grachov** [gatsbyjs/gatsby#32801] [Help]
Gatsby [generates a polyfill JS](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby/src/utils/webpack.config.js#L201) but [does not add it](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby/src/utils/worker/child/render-html.ts#L131) to the page in production mode. Is it expected?

**Comment by herecydev** [comment]
It's added as a `nomodule` script to each page if you are targeting browsers which require it. You should be able to verify this.

**Reply by grachov** [reply]
It should in theory according to [these lines](https://github.com/gatsbyjs/gatsby/blob/58d5a2c6955f1263dd5f2b28369a9c177485d36a/packages/gatsby/cache-dir/static-entry.js#L413) but I have `“polyfill”: []` in my chunk mapping so it does not work.

**Reply by grachov** [reply]
Ok, it’s probably an issue on my side. I’ve checked this functionality on new empty website and it seems to be working fine.
